.github/workflows/win.yml

```diff
@@ -5,7 +5,7 @@ on: [push, pull_request]
 jobs:
   linux-job:
     runs-on: ubuntu-latest
-    container: crystallang/crystal:0.34.0-build
+    container: crystallang/crystal:0.35.1-build
     steps:
       - name: Download Crystal source
         uses: actions/checkout@v2
```

Comment:
Line 19 can be removed now

https://github.com/crystal-lang/crystal/blob/5f9c848d85948fa72d114f8e22a3e2c99819406b/.github/workflows/win.yml#L19